api/tests/integration/users/test_init_config.py

```diff
@@ -24,3 +29,62 @@ def test_returns_200_when_no_user_exists(db, django_client):
 
     # Then
     assert response.status_code == status.HTTP_200_OK
+
+
+def test_valid_request_creates_a_superuser(db, django_client):
+    # Given
+    url = reverse("api-v1:users:config-init")
+    user_dict = {
+        "username": "test-admin",
+        "email": "test@email.com",
+        "password": "test123",
+        "site_name": "test_site",
+        "site_domain": "test.com",
+    }
+
+    # When
+    response = django_client.post(url, data=user_dict)
+
+    # Then
+    assert response.status_code == status.HTTP_200_OK
+    assert FFAdminUser.objects.filter(email=user_dict.get("email")).count() == 1
+
+
+def test_valid_request_updates_site_model(db, django_client):
```

Comment:
Can we merge this into the test above and just check that both the user and the site model get updated